src/DB_Command.php

```diff
@@ -813,18 +814,37 @@ public function size( $args, $assoc_args ) {
 			}
 		} else {
 
-			// Get the database size.
-			$db_bytes = $wpdb->get_var( $wpdb->prepare(
-				"SELECT SUM(data_length + index_length) FROM information_schema.TABLES where table_schema = '%s' GROUP BY table_schema;",
-				DB_NAME
+			// Display all tables size information.
+			if ( $all_tables ) {
```

Comment:
There's no need for a special case handling here. In the above condition `if ( $tables ) {}`, you can just add the other case as well, it should work with the existing logic:
`if ( $tables || $all-tables || $all-tables-with-prefix ) {}`